tests/test_sradb.py

```diff
@@ -0,0 +1,22 @@
+"""Tests for sradb.py
+"""
+
+import os
+import pytest
+from pysradb import SRAdb
+from pysradb.filter_attrs import guess_cell_type, guess_tissue_type, guess_strain_type
+from sqlite3 import OperationalError
+
+def test_not_valid_file():
+    """Test to check for error if file is either not
+        present or not a valid sqlite file"""
+    path = 'SRAmetadb.sqlite' 
+    try:
+        db = SRAdb(path)
+        assert False
+    except SystemExit:
```

Comment:
In case if a file does not exist, `sqlite3.connect()` (line 27) in basedb.py will throw an Operational error which will be caught in `except:` statement (line 152 in sradb.py) which will eventually lead to a `sys.exit(1)` on line 157 in sradb.py. This is the `SystemExit` that my function would catch.